### PR TITLE
Style fixes

### PR DESCRIPTION
**Pull request by dmwinslow**
This just implements a number of small style fixes that came up when reviewing the code with @hydrocrust and @kristindickerson . None of these should actually change what the code does at all, they're just to aid readability.

### CODE REVIEW

File: fehm_toolkit/heat_in.py [hunk 1]
```diff
@@ -98,11 +94,13 @@ def _plot_heaflux_3d(plot_data: pd.DataFrame):
...
-        if -1 not in region:
-            polygon = [vor.vertices[i] for i in region]
-            ax.fill(*zip(*polygon), color=mapper.to_rgba(heatflux_mW[r]))
+    for region_index, region_heatflux_mW in zip(vor.point_region, heatflux_mW):
+        region = vor.regions[region_index]
+        if -1 in region:
+            continue
+
+        polygon = [vor.vertices[i] for i in region]
+        ax.fill(*zip(*polygon), color=mapper.to_rgba(region_heatflux_mW))
```
Comment (dmwinslow) [inline fehm_toolkit/heat_in.py:103]: This change removes the old-style loop over an integer and instead pulls items out of the two lists one at a time directly - which is generally considered more "Pythonic".

File: fehm_toolkit/heat_in.py [hunk 2]
```diff
@@ -13,10 +13,6 @@
 
 logger = logging.getLogger(__name__)
 
-HEATFLUX_INPUT_ZONE = 'bottom'
-HEATFLUX_HEADER = 'hflx\n'
-HEATFLUX_FOOTER = '0\n'
-
```
Comment (dmwinslow) [inline fehm_toolkit/heat_in.py:19]: I realised after chatting with Kristin about this that these variables are only used in one place - I think it's clearer just putting them in-line directly.

File: fehm_toolkit/heat_in.py (hunk 1 above)
Comment (hydrocrust) [reply fehm_toolkit/heat_in.py:103]: Don't know this structure, but I'll learn!

File: fehm_toolkit/heat_in.py (hunk 2 above)
Comment (hydrocrust) [reply fehm_toolkit/heat_in.py:19]: Aha, good point. Perhaps they will be used elsewhere at a later time.

**Review by hydrocrust — COMMENTED** [review]
Thanks.

**Pull request merged**